verify-sna/assets/script.js

```diff
@@ -0,0 +1,119 @@
+const verificationsStatusSpan = document.getElementById('verifications-status');
```

Comment:
Can't we call this `utils.js` instead of `script.js`?

Reply:
Sure sir, I'll change the file name